app/src/main/java/com/example/ithaca_transit_android_v2/Models/DataLocation.kt

```diff
@@ -0,0 +1,11 @@
+package com.example.ithaca_transit_android_v2.Models
+
+import com.example.ithaca_transit_android_v2.LocationAdapter
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+// This class is used to store all of the Location objects that come from backend
+@JsonClass(generateAdapter = true)
+class DataLocation(
+    @Json(name = "data")
+    val data: List<LocationAdapter.JsonLocation>
+)
```

Comment:
This is repeated code

Reply:
regardless of if we keep `Coordinate`, this class is required to parse the `data` attribute of  backend's response `{"success": ..., "data": ...}`into the objects that we want. this wouldn't be a problem if we don't receive `success` in the body of the response